src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -52,6 +54,28 @@ public class MetricHelper
         /// </summary>
         private readonly object _lock = new object();
 
+        private static string _hashMacAddress = string.Empty;
+
+        private static string HashMacAddress
+        {
+            get
```

Comment:
resolved offline - let's not test the value of _hashMacAddress in the code that sets the value.